### DISCUSSION TITLE
proxy error [('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')]

### DISCUSSION THREAD

**Discussion by liqi6811** [ranaroussi/yfinance#1629] [Q&A]
Hi team, 

I am using company laptop to extract price data. my code is as below: 

msft = yf.Ticker("MSFT")
print(msft)
msft.history(period="max", proxy="https://company_proxy_ip:8080")

but I got error: [('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')]
so, I tried below code: 

msft = yf.Ticker("MSFT")
print(msft)
msft.history(period="max", proxy="https://company_proxy_ip:8080", verify=False)

but still the same error, how to not to verify certificate? 

Thank you, 
Ling

**Comment by Neurobaby** [comment]
I had an SSL error and this solution worked:

https://stackoverflow.com/questions/59642580/ssl-error-when-trying-to-fetch-stock-data-using-pandas-datareader
